Backport of:

From 363056f7db6f61f818523888085638e85c6a81f7 Apr, 2 2017
Description: Use NORMAL priority for SSLv23_*_method.  Instead of
 enforcing TLS1.0/SSL3.0 use gnutls NORMAL priority for SSLv23_*_methods.
Author: Andreas Metzler <ametzler@bebt.de>
Last-Update: 2017-04-02
Bug-Ubuntu: https://launchpad.net/bugs/1709193
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=857436

--- gnutls28-3.4.10.orig/extra/gnutls_openssl.c
+++ gnutls28-3.4.10/extra/gnutls_openssl.c
@@ -483,7 +483,7 @@ SSL_METHOD *SSLv23_client_method(void)
 		return NULL;
 
 	strcpy(m->priority_string,
-	       "NONE:+VERS-TLS1.0:+VERS-SSL3.0:+CIPHER-ALL:+COMP-ALL:+RSA:+DHE-RSA:+DHE-DSS:+MAC-ALL");
+	       "NORMAL");
 
 	m->connend = GNUTLS_CLIENT;
 
@@ -498,7 +498,7 @@ SSL_METHOD *SSLv23_server_method(void)
 		return NULL;
 
 	strcpy(m->priority_string,
-	       "NONE:+VERS-TLS1.0:+VERS-SSL3.0:+CIPHER-ALL:+COMP-ALL:+RSA:+DHE-RSA:+DHE-DSS:+MAC-ALL");
+	       "NORMAL");
 	m->connend = GNUTLS_SERVER;
 
 	return m;
